sale_report_delivered/tests/test_sale_report_delivered.py

```diff
@@ -83,4 +89,4 @@ def test_sale_report_delivered_read_group(self):
             ],
             groupby=["order_id"],
         )
-        self.assertEqual(res[0]["margin_percent"], 100)
+        self.assertEqual(round(res[0]["margin_percent"], 2), 100.00)
```

Comment:
Better use `assertAlmostEqual` for rounding issues

Reply:
Changed